virttest/utils_test/libvirt.py

```diff
@@ -55,6 +55,7 @@
 from virttest import utils_libvirtd
 from virttest import utils_config
 from virttest import utils_split_daemons
+from virttest.remote import run_remote_cmd
```

Comment:
Do you mind using a similar import strategy like the rest of the modules, i.e.
```
from virttest import remote as remote_old
...
then use `remote_old` to call the `run_remote_cmd` instead?

The reason for this is that we can easily trace what remote functionality is left in virttest and still not migrated and later on more easily rename `virttest.remote` to `vremote` or another simpler name which is already planned.

Reply:
Good to know, that makes sense. Updated.